tests/IntegrationTests/TableView.fs

```diff
@@ -0,0 +1,57 @@
+module Pulsar.Client.IntegrationTests.TableView
+
+open System
+open System.Linq
+open Expecto
+open Expecto.Flip
+open Pulsar.Client.Api
+
+open Serilog
+open Pulsar.Client.IntegrationTests.Common
+open FSharp.Control
+
+[<Tests>]
+let tests =
+    testList "TableView" [
+        testAsync "TableView with non-partition-topic works fine" {
```

Comment:
When I use the `testTask`, throw the following log:
```
Lookup on object of indeterminate type based on information prior to this program point. A type annotation may be needed prior to this program point to constrain the type of the object. This may allow the lookup to be resolved.
```
```
No overloads match for method 'Bind'.

Known types of arguments: Threading.Tasks.Task<ITableView<byte[]>> * (('a list -> 'b) -> Tasks.CopiedDoNotReference.TaskBuilder.Step<unit>) when 'b: null
```

I cannot use `tableView.Count`, could you have any ideas?

Reply:
The F# compile order is disgusting :(

Reply:
Sorry, I'll be able to only try it in the evening today. In the meantime you can look at other tests - we transitioned to testTask everywhere just several days ago